src/Integration.Vsix/Analysis/UserSettingsProvider.cs

```diff
@@ -33,7 +33,7 @@ internal sealed class UserSettingsProvider : IUserSettingsProvider, IDisposable
         // Note: the data is stored in the roaming profile so it will be sync across machines
         // for domain-joined users.
         public static readonly string UserSettingsFilePath = Path.GetFullPath(
-            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "SonarLint for Visual Studio", "settings.json"));
+            Path.Combine(EnvironmentVariableProvider.Instance.GetSLVSAppDataRootPath(), "settings.json"));
```

Comment:
I've changed this property to use the new extension method and manually tested the result.
Refactoring the code further was out of the scope of this commit (by this point, I was beginning to wish I had just added the new wrapper and extension method, and created a separate ticket to handle refactoring the existing code at some point in the future).